docs/troubleshooting.md

```diff
@@ -55,62 +161,22 @@ to ensure that the required GAC installations are updated.
 For more information about the GAC usage by the automatic instrumentation,
 see [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1906#issuecomment-1376292814).
 
-## Assembly in AdditionalDeps was not found
+### Assembly in AdditionalDeps was not found
 
-When you get an error message starting with
+#### Symptoms
 
-```terminal
-An assembly specified in the application dependencies manifest (OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json) was not found
-  ...
+Error message similar to the one below:
+
+```txt
+An assembly specified in the application dependencies manifest (OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json) was not found  
 ```
 
-you may have hit one of the known issues with this symptom:
+#### Related issues
 
 - [#1744](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1744)
-  Instrumentation does not work with dotnet CLI
 - [#2181](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2181)
-  Add arm64 architecture to the shared store - this affects Apple M1 machines.
 
-If it is not one of the issues above [host tracing](https://github.com/dotnet/runtime/blob/edd23fcb1b350cb1a53fa409200da55e9c33e99e/docs/design/features/host-tracing.md#host-tracing)
-should be used to gather the information needed to investigate the problem.
-Please, set the following environment variables:
+#### Solution
 
-```terminal
-COREHOST_TRACE=1
-COREHOST_TRACEFILE=corehost_verbose_tracing.log
-```
-
-and re-run the application to collect the log.
-
-## High CPU usage
-
-Make sure that you have not enabled the automatic instrumentation globally
-by setting the environment variables at system or user scope.
-
-If the system or user scope is intended, use the `OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES`
-environment variables to exclude applications from the automatic instrumentation.
-
-## Collect debug logs
-
-Detailed debug logs can help you troubleshoot instrumentation issues, and can be
-attached to issues in this project to facilitate investigation.
-
-To get the detailed logs from the OpenTelemetry .NET Automatic Instrumentation, set
-the `OTEL_LOG_LEVEL` environment variable to `debug` before the
-instrumented process starts.
-
-By default, the library writes the log files under predefined locations. If needed,
-change the default location by updating the `OTEL_DOTNET_AUTO_LOG_DIRECTORY`
-environment variable.
-
-After obtaining the logs, remove the `OTEL_LOG_LEVEL`
-environment variable to avoid unnecessary overhead.
-
-## Nothing happens
-
-It may occur that the .NET Profiler is unable to attach
-and therefore no logs would be emitted.
-
-The most common reason is that the instrumented application
-has no permissions to load the OpenTelemetry .NET Automatic Instrumentation
-assemblies.
+If issue you encountered is not one of the issues above, see general steps
+at the top to collect additional info to help facilitate issue investigation.
```

Comment:
```suggestion
If you encounter an issue not listed on this page, see [General steps](#general-steps)
to collect additional diagnostic information. This might help facilitate troubleshooting.
```

Would it be useful to link to issue guidelines?